src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -94,12 +94,10 @@ Generic fields MUST be present:
 
 {{ MACROS___make_metadata_table(
    {
-      "TaskName": "REQUIRED",
+      "TaskName": ("REQUIRED", "A RECOMMENDED convention is to name resting state task using labels beginning with `rest`."),
    }
 ) }}
 
-Note that the `TaskName` field does not have to be a "behavioral task" that subjects perform, but can reflect some information about the conditions present when the data was acquired (for example, `"rest"`, `"sleep"`, or `"seizure"`).
```

Comment:
I took the liberty to remove this sentence from the iEEG page as its content is not specific to iEEG. Happy to leave it in or better add to the definitions for fMRI, iEEG, MEG, EEG.

Reply:
I think it's in the iEEG spec because of the large amount of iEEG datasets with patients not doing anything in particular (not even deliberate rest) ... for example because everybody is waiting for the next onset of a seizure. It may be that a better place for this info is in the starter kit though ...

I am not opposed to removing it here as long as we have it **somwhere**

Reply:
hum... that makes sense that for this use case this should be emphasized here. Let's keep it in for now.

Reply:
Agree with reverting this deletion. Otherwise +1 to merge.